cdds/cdds/misc/system_information.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# (C) British Crown Copyright 2025, Met Office.
+# Please see LICENSE.md for license details.
+import os
+
+
+def check_environment_variables():
+    """
+    Print current values of the required environment variables.
+    Notify user of any that are currently unset or any invalid paths that have been assigned.
+    """
+    required_vars = [
+        'CYLC_VERSION',
+        'TZ',
+        'CDDS_PLATFORM',
+        'CDDS_ETC',
+        'CDDS_ENV_COMMAND',
+    ]
+
+    required_path_vars = [
+        'CDDS_ETC',
+        'CDDS_ENV_COMMAND'
+    ]
+
+    env_vars = dict(os.environ)
+    unset_vars = []
+    unresolved_paths = []
+
+    for var in required_vars:
+        x = env_vars.get(var)
+        print(f"{var}: {x}")
+        if not x:
+            unset_vars.append(var)
```

Comment:
I would prefer to avoid use of single character variable names except as loop variables, `var` is also a little close to `vars()` a built in function.